server/conf/routes

```diff
@@ -91,6 +91,7 @@ POST    /admin/questions/:id/restore controllers.admin.AdminQuestionController.r
 POST    /admin/questions/:id/archive controllers.admin.AdminQuestionController.archive(request: Request, id: Long)
 # For rendering North Star question previews on the admin question page
 GET     /admin/preview/:questionType controllers.admin.NorthStarQuestionPreviewController.sampleQuestion(request: Request, questionType: String)
+GET     /admin/preview/card/:programId controllers.admin.NorthStarProgramCardPreviewController.cardPreview(request: Request, programId: Long)
```

Comment:
(blocking) Please clean up the alignment between the url path and the controller action. Add space for better readability.

Reply:
Sure, how does this look? Is there a recommended amount of space?